### ISSUE TITLE
Menu positioning not correct on different taskbar positions on Windows

### ISSUE THREAD

**Issue by vkolova** [maxogden/menubar#232]
[//]: # "Hey there and thank you for using Menubar's issue tracker!"
[//]: # "Please fill in these information to describe your issue."

### Description
Positioning of menu window is incorrect on Windows when you change taskbar position.

### Steps to Reproduce the Problem

1. Add menubar
2. Change taskbar position

### Actual Behaviour

(wait for images to load, they're pretty large files.)

![top](http://prikachi.com/images/596/9592596J.png)
![left](http://prikachi.com/images/598/9592598e.png)
![right](http://prikachi.com/images/599/9592599H.png)

### Specifications

- Menubar version: 6.1.0
- Platform: Windows 10
- Electron version: 5.0.1

**Comment by amaurymartiny** [comment]
Agreed, reproduced. Since I think changing taskbar position is not a frequent operation, I consider this issue a lower-priority.

If anyone wants to work on this, feel free to open a PR!

**Comment by thomasdarde** [comment]
Hello

It's  not perfect but this was wat I did put in place

```
mb.on("show", function() {
    if(process.platform === 'win32'){
      var trayBounds = mb._tray.getBounds()
      var windowPosition = "trayBottomCenter"
      if(trayBounds.width === trayBounds.height && trayBounds.x < trayBounds.y) {
        log.info("Tray is on left side")
        windowPosition = "center"
      }
      // Force right place for the window
      // On launch it can show beside the menubar
      mb.setOption("windowPosition", windowPosition)
    }
  })
```

**Comment by rwwagner90** [comment]
@thomasdarde does that work for both left positioned taskbars and the normal bottom position? If so, is this something we could incorporate into the library?

**Comment by thomasdarde** [comment]
it does, feel free to reuse this code

**Comment by rwwagner90** [comment]
@thomasdarde would you perhaps be able to open a PR? This was the most recently merged one for Windows placement https://github.com/maxogden/menubar/pull/307/files. Perhaps you could use that as reference for where to put in your solution?

**Comment by thomasdarde** [comment]
sorry it's a long time since I did update this code and I'm quite busy now
